src/lib/hooks/useExercise.ts

```
@@ -0,0 +1,36 @@
+    return {
+      exercise: null,
+      isLoading: false,
+      error: new Error('Exercise ID is required'),
+    };
+  }
+
+  const {
+    data: exercise,
+    isLoading,
+    error,
+  } = useQuery<Exercise, Error>({
```

ESLint

[error] React Hook "useQuery" is called conditionally. React Hooks must be called in the exact same order in every component render. Did you accidentally call a React Hook after an early return?

(react-hooks/rules-of-hooks)